app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -85,8 +85,12 @@ def supported_api_versions_from_sdk
     probe_results = OvirtSDK4::Probe.probe(probe_args)
     probe_results.map(&:version) if probe_results
   rescue => error
-    _log.error("Error while probing supported api versions #{error}")
-    raise
+    # Note that errors when trying to find the supported API versions are perfectly normal, in particular authorization
+    # errors are expected, as in many situations, for example during discovery, the user name and the password aren't
+    # yet known. In these situations we *must* return an empty array, to indicate to the caller that it wasn't possible
+    # to determine the supported API versions.
+    _log.info("Can't determine the API versions supported by the server: #{error}")
+    []
```

Comment:
we won't have information why the probing failed. In situations like wrong credentials we won't state what happened. Am I right?

Reply:
We won't have information on why determining the supported versions failed. But that is just one part of the process of checking the credentials. After determining the supported versions we send additional requests to check the credentials (if the credentials are available). Those requests will fail, and the error will be reported to the user. The net result is that the user will see a message in the UI saying that the user name or password isn't correct.

Reply:
OK, so if the user gets this information I am fine with the change.